content/installation/installing-manager.md

```diff
@@ -0,0 +1,104 @@
+
+{{% /gsNote %}}
+
+When you install Cloudify Manager, you can specify the private IP address, public IP address and administrator password as command parameters. If you use _-v_ for the cfy_manager command, you can see installation logs located at: ```/var/log/cloudify/manager/cfy_manager.log```. You can also specify installation parameters in the [config.yaml file]({{< relref "installation/installing-manager.md#changing-the-manager-parameters" >}}).
+
+{{% gsNote title="Administrator Password" %}}
+
+If you do not specify an administrator password, the installation process generates a random password and shows it as output when the installation is complete. We recommend that you specify an administrator password according to your security policy. 
+
+{{% /gsNote %}}
+
+You can install the [Cloudify CLI]({{< relref "installation/installing-cli.md" >}}) on a separate host to manage your Cloudify Manager remotely.
+
+## Installing Cloudify Manager
+
+_To install Cloudify Manager:_
+
+1. Go to the download page of the [Cloudify website](http://cloudify.co/download/).
+1. Copy the link address for the Cloudify Manager rpm file.
+1. From the CLI of your target host, run: ```sudo yum install \<rpm file link address>```
+   For example: ```sudo yum install http://cloudify-release-eu.s3.amazonaws.com/cloudify/4.3.0/release/cloudify-manager-install-4.3ga.x86_64.rpm```
+1. To change default configuration settings, edit the ```/etc/cloudify/config.yaml``` file.
+1. To install Cloudify Manager, run: ```cfy_manager install --private-ip <PRIVATE_IP> --public-ip <PUBLIC_IP>```
+
+  For more about the configure parameters, run: ```cfy_manager configure -h```
+
+## Other Installation Actions
+
+### Validating the Installation
+
+When the installation process is complete, you can use ```cfy status``` to make sure that all of the Cloudify services are running.
+Cloudify Premium customers can access the Cloudify Web Interface at: ```http://<manager_public_address>```
+
+An example output:
+{{< gsHighlight  sh >}}
+$ cfy status
+
+...
+
+Retrieving manager services status... [ip=127.0.0.1]
+
+Services:   
+   +--------------------------------+---------+
+   |            service             |  status |
+   +--------------------------------+---------+
+   | InfluxDB                       | running |
+   | Celery Management              | running |
+   | Logstash                       | running |
+   | RabbitMQ                       | running |
+   | AMQP InfluxDB                  | running |
+   | PostgreSQL                     | running |
+   | Manager Rest-Service           | running |
+   | Cloudify Stage                 | running |
+   | Webserver                      | running |
+   | Riemann                        | running |
+   | Webserver                      | running |
+   +--------------------------------+---------+
+
+   ...
+   {{< /gsHighlight >}}
+
+### Changing the Manager Parameters
```

Comment:
Something very important to note about this:
By default, when running `cfy_manager configure` the DB is not cleared. This has some ramifications (for example, the provider context is not update), but it is the default behavior to avoid scenarios where the user will delete their whole DB by mistake.
This applies to both `cfy_manager configure` and `cfy_manager install`. In order to circumvent this behavior the user can provide the `--clean-db` flag which will force the recreation of the DB (i.e. the full configuration process as performed on the initial install).

Reply:
--clean-db applies to both configure and install?

Reply:
I'll add it in another commit